charts/datadog/templates/NOTES.txt

```diff
@@ -538,3 +538,14 @@ You are using the datadog.securityAgent.compliance.xccdf.enabled parameter which
 This version still supports both but the support of the old name will be dropped in the next major version of our Helm chart.
 More information about this change: https://github.com/DataDog/helm-charts/pull/1161
 {{- end }}
+
+{{- if .Values.clusterAgent.admissionController.agentSidecarInjection.enabled }}
+    {{- if (semverCompare "<7.52.0" .Values.clusterAgent.image.tag) }}
+##############################################################################
+####               WARNING: Sidecar injection not supported.              ####
+##############################################################################
+
+The clusterAgent.admissionController.agentSidecarInjection.enabled is only supported
+by Cluster Agent 7.52.0 or later. Enabling this flag will not have any effect.
+    {{- end }}
```

Comment:
```suggestion
```

Reply:
I'll remove this in the next rev. This will be merged after agent version is bumped.

Reply:
sorry mind clarifying what you mean? (i just suggested this change because i consolidated the `if` statements above)

Reply:
oh, I mean to remove whole thing. I added, thinking this would be merged before 7.52 bump but it won't. There are no other tag checks in this file for specific features so no need to have it for this one either.